Formula/socket_vmnet.rb

```diff
@@ -0,0 +1,46 @@
+class SocketVmnet < Formula
+  desc "Daemon to provide vmnet.framework support for rootless QEMU"
+  homepage "https://github.com/lima-vm/socket_vmnet"
+  url "https://github.com/lima-vm/socket_vmnet/archive/refs/tags/v1.1.0.tar.gz"
+  sha256 "da85362518c4ccfef3587240d380d52d7c7621c4567f1d9a476e1c17c5a7563b"
+  license "Apache-2.0"
+  head "https://github.com/lima-vm/socket_vmnet.git", branch: "master"
+
+  depends_on :macos
+
+  def install
+    # make: skip "install.launchd"
+    system "make", "install.bin", "install.doc", "VERSION=#{version}", "PREFIX=#{prefix}"
+    # Create optional launchd files
+    mkdir_p "#{prefix}/launchd"
+    Pathname.glob("launchd/*") do |file|
+      inreplace file, "/opt/socket_vmnet", opt_prefix
+      cp file, "#{prefix}/launchd/#{file.basename}"
+    end
+  end
+
+  def caveats
+    # #{HOMEBREW_PREFIX}/bin=/usr/local/bin is often chgrped to staff (not admin)
+    <<~EOS
+      1. The socket_vmnet binary is installed as `#{opt_prefix}/bin/socket_vmnet` .
+         This binary has to be executed as the root.
+         The binary is also symlinked into `#{HOMEBREW_PREFIX}/bin/socket_vmnet`,
+         however, it should be noted that a non-admin user might be able to replace this symlink.
+         So, it is highly recommended to use `#{opt_prefix}/bin/socket_vmnet`
+         as the binary path.
+
+      2. To install an optional launchd service, run the following commands:
+         ---
+         sudo install -o root -g wheel\
+           #{prefix}/launchd/io.github.lima-vm.socket_vmnet.plist\
+           /Library/LaunchDaemons/io.github.lima-vm.socket_vmnet.plist
+         sudo launchctl bootstrap system /Library/LaunchDaemons/io.github.lima-vm.socket_vmnet.plist
+         sudo launchctl kickstart -kp system/io.github.lima-vm.socket_vmnet
+         ---
```

Comment:
Why doesn't this work with `sudo brew services`?

Reply:
Thanks, updated to use `sudo brew services`